src/components/StyledForm.js

```diff
@@ -0,0 +1,49 @@
+import {useState} from 'react';
+import styled from 'styled-components';
+
+import useStore from '../hooks/useStore';
+
+export default function StyledForm() {
+	const [inputValue, setInputValue] = useState('');
+
+	const items = useStore(state => state.items);
+	const addItem = useStore(state => state.addItem);
+
+	return (
+		<>
+			<ul>
+				{items.map(item => {
+					return <li key={item.id}>{item.item}</li>;
+				})}
+			</ul>
+			<form
+				aria-label="Add a item"
+				onSubmit={event => {
+					event.preventDefault();
+					addItem(inputValue);
+					setInputValue('');
+				}}
+			>
+				<label htmlFor="item">
+					Item:
+					<StyledInput
+						id="item"
```

Comment:
If you nest your input element inside the label element, you do not need to connect both with the `htmlFor` and `id` attribute as they are already related to each other by nesting.
```suggestion
				<label>
					Item:
					<StyledInput
```